recipes/netcdf/all/conanfile.py

```diff
@@ -32,6 +33,7 @@ class NetcdfConan(ConanFile):
         "with_hdf5": True,
         "cdf5": True,
         "dap": True,
+        "byterange": True,
```

Comment:
it's false by default upstream: https://github.com/Unidata/netcdf-c/blob/6a935c68120623a9932cec80e7cad22abe6f6428/CMakeLists.txt#L1053

Reply:
ok